pwiz_tools/Shared/MSGraph/MSGraphPane.cs

```diff
@@ -422,7 +422,8 @@ private void drawLabels( Graphics g )
                     {
                         ZOrder = ZOrder.A_InFront,
                         FontSpec = annotation.FontSpec,
-                        IsClippedToChartRect = true
+                        IsClippedToChartRect = true,
+                        Tag = annotation.ZOrder

```

Comment:
I wonder whether we should add "Tag" to the class "PointAnnotation". I am surprised nobody has needed that yet. I wonder how other code figures out which object the user clicked on.